api/core/span_context.go

```diff
@@ -80,6 +85,11 @@ func (s SpanID) MarshalJSON() ([]byte, error) {
 	return json.Marshal(hex.EncodeToString(s[:]))
 }
 
+// Stringer Implementation of String() returns the string form of a SpanID
+func (s SpanID) String() string {
+	return string(s[:])
```

Comment:
You really do not want this to print raw bytes, we should always hex encode when stringifying.

Reply:
Done.